csmApp/Makefile

```diff
@@ -5,6 +5,8 @@ LIBRARY_IOC = csmbase
 csmbase_SRCS = csmbase.c
 csmbase_LIBS += $(EPICS_BASE_IOC_LIBS)
 
+USR_CFLAGS += /Od
+
```

Comment:
Is turning off optimisation required for it to work?

Reply:
Removed